qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/page/AttachmentsPage.java

```diff
@@ -34,6 +41,22 @@ public void setFileUrl(String fileUrl) {
 		WebElement element = driver.findElement(By.xpath("//input[@type='file']"));
 		element.sendKeys(fileUrl);
 	}
+	
+	public String createRandomFile() throws IOException {
```

Comment:
Does this need to be `public`?

Reply:
Although the file name is unguessable, it's not really a random file per se...

```suggestion
	public String createTemporaryFile() throws IOException {
```

Reply:
> Does this need to be `public`?

@ibacher would it be better if I made it private, What are your thoughts?

Reply:
@kdaud , @sherrif10 , @ibacher  I have made another commit. Please could you take some time to review it.